frontend/src/components/feature/NavBar/AddTripButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { WriteMini } from '../../../assets/ext-icon'
+
+export default function AddTripButton(props) {
+    return (
+        <button onClick={props.onClick} className='text-white pr-2 flex flex-row items-center'>
```

Comment:
```suggestion
        <button onClick={props.onClick} className='text-white pr-2 flex-row items-center'>
```

Reply:
In Tailwind CSS 
Flex is the superclass and flex-row is the class property
Definition of class is mandatory unlike css
PFL
![](https://tailwindcss.com/docs/flex-direction)